pkg/storage/unified/search/bleve.go

```diff
@@ -1167,6 +1167,23 @@ func (b *bleveIndex) toBleveSearchRequest(ctx context.Context, req *resourcepb.R
 	return searchrequest, nil
 }
 
+func removeSmallTerms(query string) string {
+	words := strings.Fields(query)
+	validWords := make([]string, 0, len(words))
+
+	for _, word := range words {
+		if len(strings.TrimSpace(word)) >= EDGE_NGRAM_MIN_TOKEN {
+			validWords = append(validWords, word)
+		}
```

Comment:
Nit - I guess we don't need the `strings.TrimSpace` as it is already handled by the `strings.Fields`.
```suggestion
		if len(word) >= EDGE_NGRAM_MIN_TOKEN {
			validWords = append(validWords, word)
		}
```